react-native-libraries.json

```
@@ -14809,5 +14809,16 @@
     "expoGo": true,
     "fireos": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/XterioTech/XterioSDK-RN/tree/main/xterio-auth",
+    "npmPkg": "@xterio-sdk/rn-auth",
+    "examples": [
+      "https://github.com/XterioTech/XterioSDK-RN/blob/main/xterio-auth/README.md"
```

```suggestion
      "https://github.com/XterioTech/XterioSDK-RN/tree/main/xterio-auth/example"
```

The package name will link to GitHub repository already, so we expect apps, project, Snacks, or website as an `example` value, see:
* https://github.com/react-native-community/directory?tab=readme-ov-file#examples

In this case, the example project exist, so we can just swap the links.